legislators-social-media.yaml

```diff
@@ -1712,6 +1712,7 @@
     thomas: '01685'
     govtrack: 400279
   social:
+    twitter: repjeffmiller
```

Comment:
Jeff Miller's [official website](http://jeffmiller.house.gov/) doesn't link to this account, but the Twitter account is verified and does link to his official website, so I think, it's :+1:.